easybuild/easyconfigs/v/VMD/VMD-1.9.4a43-fosscuda-2019b-Python-3.7.4.eb

```
@@ -0,0 +1,62 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Authors::   Stephane Thiell <sthiell@stanford.edu>
+##
+name = 'VMD'
+version = '1.9.4a43'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.ks.uiuc.edu/Research/vmd'
+description = """VMD is a molecular visualization program for displaying, animating, and analyzing large biomolecular
+ systems using 3-D graphics and built-in scripting."""
+
+toolchain = {'name': 'fosscuda', 'version': '2019b'}
+
+source_urls = [
+    'https://www.ks.uiuc.edu/Research/vmd/vmd-1.9.4/files/alpha/',
+    'https://www.ks.uiuc.edu/Research/vmd/vmd-%(version)s/files/final',
+    'http://webclu.bio.wzw.tum.de/stride/'
+]
+sources = [
+    'vmd-%(version)s.src.tar.gz',
+    {'filename': 'stride.tar.gz', 'extract_cmd': "tar -C vmd-%(version)s/lib/stride -xf %s"},
+]
+patches = [
+    ('VMD-1.9.3_plugins.patch'),
+    ('VMD-1.9.3_surf_Makefile.patch', 'vmd-%(version)s'),
+    ('VMD-1.9.3_surf_bad_printfs.patch', 'vmd-%(version)s'),
+    ('VMD-1.9.3_stride_Makefile.patch', 'vmd-%(version)s'),
+    ('VMD-1.9.3_stride_MAX_AT_IN_RES.patch', 'vmd-%(version)s'),
+    ('VMD-1.9.4_configure.patch', 'vmd-%(version)s'),
+    ('VMD-1.9.4_extra_colors.patch', 'vmd-%(version)s'),
```

@bedroge please use the full version here (and change file names):
```suggestion
    ('VMD-1.9.4a43_configure.patch', 'vmd-%(version)s'),
    ('VMD-1.9.4a43_extra_colors.patch', 'vmd-%(version)s'),
```

---

The reason I used 1.9.4 here is that they bring out quite a lot of 1.9.4a<somenumber> releases, and these patches should most likely work for all of them. So I figured that using 1.9.4 would be nicer, so it can be nicely reused. Or should future 1.9.4a releases just specify VMD-1.9.4a43_whatever.patch?

---

the latter, yes. and while it may be true that it will probably work for future 1.9.4a releases, we cannot be sure of it.

---

okay, I'll change the name